crates/router/src/routes/app.rs

```diff
@@ -1612,30 +1612,42 @@ impl BusinessProfile {
 ))]
 impl BusinessProfile {
     pub fn server(state: AppState) -> Scope {
-        web::scope("/account/{account_id}/business_profile")
+        web::scope("/account/{account_id}")
             .app_data(web::Data::new(state))
             .service(
-                web::resource("")
-                    .route(web::post().to(business_profile_create))
-                    .route(web::get().to(business_profiles_list)),
-            )
-            .service(
-                web::scope("/{profile_id}")
+                web::scope("/business_profile")
                     .service(
-                        web::resource("")
-                            .route(web::get().to(business_profile_retrieve))
-                            .route(web::post().to(business_profile_update))
-                            .route(web::delete().to(business_profile_delete)),
+                        web::scope("/account/{merchant_id}").service(
```

Comment:
Can you check this again. This is wrong. Merchant id will be there 2 times in the path.